app/src/main/res/layout/fragment_dialog_detail_character.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <ImageView
+        android:id="@+id/characterPicture"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent" />
+
+    <LinearLayout
```

Comment:
Please don't use a LinearLayout inside a relative layout, this will decrease the performance of the app. If possible, please refactor to constraintLayout. In other case, only use the RelativeLayout and remove the use of LinearLayout